MediaBrowser.Plugins.Anime/Configuration/configPage.html

```diff
@@ -43,7 +43,7 @@
                                 <label for="chkAniDB_wait_time">
                                     AniDB sleep time to prevent IP ban (ms)
                                 </label>
-                                <input id="chkAniDB_wait_time" name="chkAniDB_wait_time" type="number" value="0"/>
+                                <input id="chkAniDB_wait_time" name="chkAniDB_wait_time" type="number" value="2000"/>
```

Comment:
This should be a default in LoadConfiguration and thus shouldn't be hard coded in the HTML. Does removing value altogether work?